client/src/components/MainPage/index.jsx

```diff
@@ -19,6 +57,17 @@ class MainPage extends Component {
             />
             <ProgressBar />
           </div>
+          <div className="mainpage__content">
+            <div className="mainpage__content--header">
```

Comment:
the double hyphens are for modifiers, like color changes and such.
as long as we're in the element's name, we should only have single hyphen.
```suggestion
            <div className="mainpage__content-header">
```